src/components/common/datePicker/CorrectionCustomHeader.tsx

```diff
@@ -1,75 +1,87 @@
 import styled from '@emotion/styled';
 
-import ArrangeBtn from '../arrangeBtn/ArrangeBtn';
-import TextboxInput from '../textbox/TextboxInput';
-
-import formatDatetoString from '@/utils/formatDatetoString';
+import Button from '../v2/button/Button';
+import IconButton from '../v2/IconButton';
+import MainDate from '../v2/TextBox/MainDate';
 
 interface CustomHeaderProps {
-	prevDate: Date | null;
+	selected: Date | null;
 	decreaseMonth: () => void;
 	increaseMonth: () => void;
 	prevMonthButtonDisabled: boolean;
 	nextMonthButtonDisabled: boolean;
 	onChange: (date: Date | null) => void;
-	dateTextRef: React.RefObject<HTMLInputElement>;
+	onClose: () => void;
 }
 
 function CorrectionCustomHeader({
-	prevDate,
+	selected,
 	decreaseMonth,
 	increaseMonth,
 	prevMonthButtonDisabled,
 	nextMonthButtonDisabled,
 	onChange,
-	dateTextRef,
+	onClose,
 }: CustomHeaderProps) {
+	const selectedDate = selected || new Date();
+	const today = new Date();
 	return (
-		<div className="react-datepicker__header-custom">
-			<InfoBox>
-				<TextboxInput
-					variant="date"
-					placeholder={formatDatetoString(prevDate)}
-					onDateChange={onChange}
-					dateTextRef={dateTextRef}
+		<HeaderLayout className="react-datepicker__header-custom">
+			<IconButton iconName="IcnX" size="small" type="normal" disabled={false} onClick={onClose} />
+			<DateWrapper>
+				<MainDate
+					month={selectedDate.getUTCMonth() + 1}
+					day={selectedDate.getUTCDate()}
+					year={selectedDate.getUTCFullYear()}
 				/>
-				<div className="react-datepicker__navigation-wrapper">
-					<BtnWrapper className="react-datepicker__navigation-container">
-						<ArrangeBtn
-							color="WHITE"
-							mode="DEFAULT"
-							size="small"
-							type="left"
-							className="react-datepicker__navigation react-datepicker__navigation--previous"
-							onClick={decreaseMonth}
-							disabled={prevMonthButtonDisabled}
-							aria-label="Previous Month"
-						/>
-						<ArrangeBtn
-							color="WHITE"
-							mode="DEFAULT"
-							size="small"
-							type="right"
-							className="react-datepicker__navigation react-datepicker__navigation--next"
-							onClick={increaseMonth}
-							disabled={nextMonthButtonDisabled}
-							aria-label="Next Month"
-						/>
-					</BtnWrapper>
-				</div>
-			</InfoBox>
-		</div>
+			</DateWrapper>
+			<div className="react-datepicker__navigation-wrapper">
+				<BtnWrapper className="react-datepicker__navigation-container">
+					<Button
+						leftIcon="IcnCalendar"
```

Comment:
요기도 피그마 상이랑 달라서 수정해주세요!! 
```suggestion
						leftIcon="IcnReturn"
```

Reply:
감사합니다! 반영완료했습니다